src/visualization/recipes_makie.jl

```diff
@@ -41,8 +43,8 @@ function iplot(u, mesh::UnstructuredMesh2D, equations, solver, cache;
   menu = Makie.Menu(fig, options=menu_options)
 
   # Initialize toggle switches for viewing the mesh
-  toggle_solution_mesh = Makie.Toggle(fig, active=true)
-  toggle_mesh = Makie.Toggle(fig, active=true)
+  toggle_solution_mesh = Makie.Toggle(fig, active=plot_mesh)
+  toggle_mesh = Makie.Toggle(fig, active=plot_mesh)
```

Comment:
Does this mean that the mesh button is visible but grayed out?

Reply:
No, just that it starts on "off". I changed my mind and thought this would be an easier way to incorporate the kwarg.

Reply:
:+1: But is the button still shown even if turning it on does not do anything, e.g., for scalardata?

Reply:
No, for ScalarData it's not shown at all.